crates/turborepo-ci/src/lib.rs

```diff
@@ -46,15 +46,12 @@ impl Vendor {
     }
 
     /// Gets user from CI environment variables
-    /// We return an empty String instead of None because
-    /// the Spaces API expects some sort of string in the user field.
-    pub fn get_user() -> String {
+    pub fn get_user() -> Option<String> {
         let vendor = Vendor::infer();
 
         vendor
             .and_then(|v| v.username_env_var)
             .and_then(|v| env::var(v).ok())
-            .unwrap_or_default()
```

Comment:
Maybe keep this in case anyone is parsing summary/dry JSON and expecting the user to be present?

Reply:
It's kept on the output end, but I changed the type because I don't love default values as None in our codebase